mod-invoice-storage/examples/invoice_line_collection.sample

```diff
@@ -4,14 +4,18 @@
       "id": "e0d08448-343b-118a-8c2f-4fb50248d672",
```

Comment:
Just left a comment on Piotr's PR about reusing UUIDs... So to be fair I'll leave one here too ;)